docker/alias/alias_computation.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""OSV alias computation."""
+import datetime
+import logging
+
+from google.cloud import ndb
+
+import osv
+import osv.logs
+
+ALIAS_GROUP_VULN_LIMIT = 32
```

Comment:
32 is quite large (for reference, we currently only have 16 data sources across all ecosystems)
Maybe take it down to 16 or so?